tools/update-webpage.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+# Copyright 2015-2016 Samsung Electronics Co., Ltd.
+# Copyright 2016 University of Szeged
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ $# -ne 1 ]; then
+  echo "Please, specify your gh-pages clone directory: update-webpage <gh-pages clone dir>"
+  exit 1
+fi
+
+gh_pages_dir=$1
+docs_dir="../docs"
```

Comment:
I'm not sure that if we ran it like this way: ``` tools/update-webpage.sh <path>``` the script will find the docs_dir.